schedulers-tests/src/test/java/org/mule/test/integration/schedule/PollScheduleNotificationTestCase.java

```diff
@@ -52,8 +57,21 @@ public void addServiceConfigurator(ServiceConfigurator serviceConfigurator) {
         // Nothing to do
       }
     });
-  }
+    builders.add(new AbstractConfigurationBuilder() {
 
+      @Override
+      protected void doConfigure(MuleContext muleContext) throws Exception {
+        ExtensionManager extensionManager;
```

Comment:
this code is repeated. Already saw this in the other PR. Anyway this can be reused? Why not just adding this into the base AbstractMuleContextTestCAse